src/controllers/verify.ts

```diff
@@ -173,23 +174,27 @@ export default class VerifyController {
   async farcasterCompact(
     @Ctx() ctx: Context,
     @Body({ required: true })
-    { address }: FarcasterVerifyBody
+    { address, ownerAddresses }: FarcasterLargerAnonymitySetVerifyBody
   ) {
     if (!(await isAddressConnectedToFarcaster(address)))
```

Comment:
There is no use in increasing the anonymity set if you only check the exposed address